docker/superset/superset_config.py

```diff
@@ -116,4 +116,24 @@ class CeleryConfig:
     # "sk": {"flag": "sk", "name": "Slovak"},
     # "sl": {"flag": "si", "name": "Slovenian"},
     # "nl": {"flag": "nl", "name": "Dutch"},
-}
\ No newline at end of file
+}
+
+# EMAIL
+
+MAIL_SERVER = 'smtp.gmail.com'
```

Comment:
So in the end you opted for gmail?

Reply:
it should be defined in the .env file
for now I am using the gmail just in order to have it ready as soon as possible
it seems that gmail will change some security options and maybe we will need to check some other alternatives as well

to summarize the free alternatives:
- sendinblue maybe would work .. but it depends on their approval in order to use the service. first time I used it was approved (using my account) but second  time it was declined (using the epigraphhub account).
- mailgun looks easy to configure, but it allows to send message just to pre-registered email (that doesn't work for us)
- mailjet looks nice, but not sure yet about the restrictions after the free trial